src/libraries/System.Private.Xml/src/System/Xml/Dom/XmlNodeReader.cs

```diff
@@ -1528,7 +1528,7 @@ private bool Read(bool fSkipChildren)
                 FinishReadBinary();
             }
 
-            bool bRead = false;
+            bool bRead;
```

Comment:
The local can either be declared below where it's assigned, or removed entirely and just have the if clause that uses it be `if (ReadNextNode(fSkipChildren))`

Reply:
Done